src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
```

Comment:
suggestion

          import { ApplicationConfig, enableProdMode } from '@angular/core';

          Explantion: Including enableProdMode is important for production setups to ensure better performance.